pkg/util/errors/wrapper.go

```diff
@@ -0,0 +1,22 @@
+package errors
+
+import (
+	"fmt"
+	"runtime"
+)
+
+func getCallerInfo() (string, int, string) {
+	pc, file, line, ok := runtime.Caller(2)
```

Comment:
if I am correct, this is relies on the place in the stack where this is called, maybe add a comment that explains that in the usage

Reply:
some comments have been added

Reply:
LGTM